pages/SignIn.js

```diff
@@ -0,0 +1,39 @@
+import BasePage from '../pages/BasePage';
+
+class SignIn extends BasePage {
+
+  //WebElements
+  get username(){ return $('[name="username"]') }
+  
+  get password() {return $('[name="password"]')}
+
+  get submitbtn() {return $('#submit')};
+
+  get text() {return $('span')};
```

Comment:
This is not a descriptive name for this element. Text is too generic. Rename it so something that gives more hints towards what is this representing for the user perspective. Also, Is this the only `span` on the page? This selector is not specific enough, as `span` is a commonly used tag for representing widely different things on html. There is a high risk of this selector breaking if any other span is added to the page in the future. Prefer a more robust selector for locating this element (something that uses their ids or a class, or another attribute that uniquely identifies it)